app/services/like_service.rb

```
@@ -4,7 +4,8 @@ def create(params, user)
     Like.create!(params)
   end
 
-  def destroy(id)
-    Like.destroy(id)
+  def destroy(id, params, current_user)
+    like = Like.find_by(likable_id: id, user_id: current_user.id, likable_type: params[:likable_type])
```

Please use a bang `!` after find_by to throw an error if like is not found